app/main.py

```diff
@@ -1,2 +1,51 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.items = []
+        self.capacity = 8
+        self.hash_table = [[] for _ in range(self.capacity)]
+
+    def __setitem__(self, key, value):
+        self.items.append([key, value])
+        threshold = int((self.capacity * 2 / 3))
+        size = len(self.items)
+        if size == threshold:
+            self.capacity *= 2
+            self.hash_table = [[] for _ in range(self.capacity)]
+            for element in self.items:
+                self.hashing(hash(element[0]) % self.capacity,
+                             element[1],
+                             element[0])
+            return
+
+        hash_ = hash(key) % self.capacity
+        self.hashing(hash_, value, key)
+
+    def __getitem__(self, item):
+        for element in self.hash_table:
```

Comment:
Why use for loop here
When you don't know the number of iterations while is a more common solution